src/Type/Php/SprintfFunctionDynamicReturnTypeExtension.php

```diff
@@ -130,10 +139,10 @@ public function getTypeFromFunctionCall(
 
 					$singlePlaceholderEarlyReturn = $checkArgType->toString();
```

Comment:
I wonder this line is also missing lowercase handling

Reply:
I shouldn't add lowercase handling here and just rely on `toString`.

I'll open a PR to update `IntegerType::toString`.